packages/action-button/src/spectrum-action-button.css

```diff
@@ -315,7 +315,6 @@ governing permissions and limitations under the License.
     line-height: var(--spectrum-actionbutton-height);
     font-size: var(--spectrum-actionbutton-font-size);
     white-space: nowrap;
-    color: inherit;
```

Comment:
This change doesn't match up with any changes from the referenced CSS PR. Is that a cause for concern?

Reply:
It's a fallback value that was injected previously by the settings SWC uses for stylelint but it's not necessary since the browser support for custom properties with fallbacks is well supported. It's a great call-out though.

Reply:
Thanks for the explanation!